docs/analyzer/decision_process.md

```diff
@@ -1,3 +1,80 @@
-# Presidio analyzer - Decision process
+# The Presidio-analyzer decision process
 
-TODO: copy from V1 interpretability.md
+## Background
+
+Presidio-analyzer traces its decision process, which allows you to investigate a specific api request, by exposing a `correlation-id` as part of the api response headers.
+
+The decision process traces explain why a specific PII was detected. For example: Which recognizer detected the entity, which regex / ML model were used, which context words improved the score, etc.
+
+## How it works
+
+The decision process can either be returned together with the Presidio-analyzer `/analyze` response, or logged into a dedicated logger.
+
+### Getting the decision process as part of the response
+
+The decision process result is added to the response automatically.
+To enable/disable it, call the `analyze` method with `remove_interpretability_response` as False and True, respectively.
+
+### Logging the decision process
+
+Each trace contains a `correlation-id` which correlates to a specific api request. The api returns a `x-correlation-id` header which you can use to the `correlation-id` and query the `stdout` logs.
```

Comment:
Not sure about my remark, but this sentence is a little not clear **TO ME**. I think something like will be a little more clean WDYT?:
Each api request contains `correlation-id` which is the trace identification. It will help you to query the `stdout` logs.
To get the id, each API response returns `x-correlation-id` header.

Reply:
Fixed, thanks!